scout/constants/case_tags.py

```diff
@@ -2,7 +2,7 @@
 
 # keys are used in the load report cli, while key_name is saved in case database documents
 CUSTOM_CASE_REPORTS = {
-    "delivery": {"key_name": "delivery_report", "format": "HTML", "pdf_export": True},
+    "delivery_report": {"key_name": "delivery_report", "format": "HTML", "pdf_export": True},
```

Comment:
🙄

Reply:
Ok, I'd rather keep that name and change the template, wait for the review

Reply:
Naa it's more comoplicated. I'd like to keep it like this instead